atm.js

```diff
@@ -1,21 +1,105 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance = 0;
+var savingsBalance = 0;
 
```

Comment:
I've seen this alot, and while its fine for this exercise. I would say that its not entirely modular. Imagine we have the ability to persist data and that we could save the state of the balances. If we instantiate the balances to 0 every time in our JS it would do just that despite what the balance is saved at.